test/e2e/util_test.go

```diff
@@ -1112,3 +1112,34 @@ func waitForLBAnnotation(t *testing.T, ic *operatorv1.IngressController, expecte
 		t.Fatalf("error updating the %q service: %v", controller.LoadBalancerServiceName(ic), err)
 	}
 }
+
+// getIngressControllerLBAddress waits for and returns the address (hostname or IP) for an
+// IngressController's load-balancer type service. It will call t.Fatal if the address is never found.
+func getIngressControllerLBAddress(t *testing.T, ic *operatorv1.IngressController) string {
+	var lbAddress string
+	err := wait.PollUntilContextTimeout(context.Background(), 5*time.Second, 3*time.Minute, false, func(ctx context.Context) (bool, error) {
```

Comment:
Is 3 minutes enough time to provision an ELB, say, 99.9% of the time?  I feel like 10 minutes would make the test less likely to flake out.